test/Microsoft.TestPlatform.AcceptanceTests/PostProcessingTests.cs

```diff
@@ -14,6 +14,8 @@
 using Microsoft.VisualStudio.TestPlatform.Utilities.Helpers;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
+# nullable disable
```

Comment:
Was there any issue without this?

Reply:
Yes, stdOut and and other variables started to fail the build, even though this code was not introduced in this PR.